datasource/hcp-packer-image/data.go

```diff
@@ -33,6 +34,9 @@ type Config struct {
 	CloudProvider string `mapstructure:"cloud_provider" required:"true"`
 	// The name of the cloud region your image is in. For example "us-east-1".
 	Region string `mapstructure:"region" required:"true"`
+	// The specific Packer builder or post-processor used to create the image.
```

Comment:
I don't think this can be a post-processor with the current limitation with have of keeping metadata just for builders. 
```suggestion
	// The specific Packer builder used to create the image.
```